test/e2e/tests/chain-interactions.spec.js

```diff
@@ -99,7 +100,9 @@ describe('Chain Interactions', function () {
         await driver.switchToWindow(extension);
 
         // verify current network
-        const networkDisplay = await driver.findElement('.network-display');
+        const networkDisplay = await driver.findElement(
+          '[data-testid="network-display"]',
+        );
```

Comment:
Nothings changed here, this is the same element as the original, just using a different selector